EMR/EMRClusterWithAdditionalSecurityGroups.json

```diff
@@ -28,8 +28,15 @@
         },
         "SubnetID": {
             "Description": "Must be a valid public subnet ID",
-            "Type": "String",
-            "Default": "subnet-dba430ad"
+            "Type": "String"
+        },
+        "AdditionalSlaveSecurityGroups": {
```

Comment:
Let's go with different language here. "Primary/Secondary" or "Main/Replica", something like that.

Reply:
@ericzbeard I have modified the parameters as below. Please review.
 
```
 AdditionalCoreNodeSecurityGroups:
    Description: Comma-delimited list of additional security groups for core and task nodes
    Type: CommaDelimitedList
  
  AdditionalPrimaryNodeSecurityGroups:
    Description: Comma-delimited list of additional security groups for primary nodes
    Type: CommaDelimitedList
```

Reply:
Unfortunate that the resource itself still has those names. Nothing you can do about that.